src/commcare_cloud/ansible/roles/elasticsearch/templates/config/elasticsearch.yml.j2

```
@@ -98,3 +135,16 @@ node.master: {{ elasticsearch_master|default('false') }}
 {% endif %}
 
 node.data: {{ elasticsearch_data|default('true') }}
+{% endif %}
+
+{% if elasticsearch_version == '7.9.1'%}
+{% if elasticsearch_data|default('true') and elasticsearch_master|default('false') %}
```

use bools not strings
```suggestion
{% if elasticsearch_data|default(True) and elasticsearch_master|default(False) %}
```